src/lib/Rule/ChangeArgumentTypeRector.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Rector\Rule;
+
+use Ibexa\Rector\Rule\Configuration\ChangeArgumentTypeConfiguration;
+use PhpParser\Node;
+use PhpParser\Node\Expr\Variable;
+use PhpParser\Node\Param;
+use PhpParser\Node\Stmt\ClassMethod;
+use PHPStan\Type\ObjectType;
+use PHPStan\Type\ObjectWithoutClassType;
+use PHPStan\Type\Type;
+use Rector\Contract\Rector\ConfigurableRectorInterface;
+use Rector\Exception\ShouldNotHappenException;
+use Rector\PHPStanStaticTypeMapper\Enum\TypeKind;
+use Rector\Rector\AbstractRector;
+use Rector\StaticTypeMapper\StaticTypeMapper;
+
+/**
+ * @see \Ibexa\Rector\Tests\Rule\ChangeArgumentTypeRector\ChangeArgumentTypeRectorTest
+ */
+final class ChangeArgumentTypeRector extends AbstractRector implements ConfigurableRectorInterface
```

Comment:
Naming: this still changes any type to `object`, right? Then rather `ChangeArgumentTypeToObjectTypeRector`.